### PR TITLE
Adding authorships and fixing missed authorship

### PR DESCRIPTION
**Pull request by gusbemacbe**
- Added my authorship
- Added missed @ahmadawais
- Sorted the list in alphabetical order
- Changing from *new style* to *new styles* and *new language* to *new languages* because new language is singular and is only one.

### CODE REVIEW

**Comment by marcoscaceres** [comment]
As I mentioned in the other PR, please avoid sorting names, as it makes a bit of a mess for all the other pull requests.

File: CHANGES.md [hunk 1]
```diff
@@ -1,36 +1,40 @@
...
-- *isbl editor light* by [Dmitriy Tarasov][]
+New styles:
+- *An Old Hope* by [Gustavo Costa][]
+- *Atom One Dark Reasonable* by [Gidi Meir Morris][]
 - *isbl editor dark* by [Dmitriy Tarasov][]
+- *isbl editor light* by [Dmitriy Tarasov][]
 - *Lightfair* by [Tristian Kelly][]
 - [*Nord*][nord-highlightjs] by [Arctic Ice Studio][]
-- *Atom One Dark Reasonable* by [Gidi Meir Morris][]
+- *Shades of Purple* by [Ahmad Awais][]
```
Comment (ahmadawais) [inline CHANGES.md]: Can you change this like this

```diff
- *Shades of Purple* by [Ahmad Awais][]
+ *[🦄 Shades of Purple](https://github.com/ahmadawais/Shades-of-Purple-HighlightJS)* by [Ahmad Awais][https://github.com/AhmadAwais]
```

Comment (ahmadawais) [reply CHANGES.md]: Looks like it's merged already so I'll send a PR.

Comment (marcoscaceres) [reply CHANGES.md]: Thanks

**Pull request merged**